### PR TITLE
feat: drop support of symfony < 4.4. add support of symfony 6

### PR DESCRIPTION
**Pull request by Chris53897**
See: https://github.com/mybuilder/cronos/pull/27
#41

Not tested in real life app.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
         "mybuilder/cronos": "~3.0",
         "doctrine/annotations": "1.*",
-        "symfony/console": "~3.0||^4.0||^5.0",
-        "symfony/config": "~3.0||^4.0||^5.0",
-        "symfony/dependency-injection": "~3.0||^4.0||^5.0",
-        "symfony/framework-bundle": "~3.0||^4.0||^5.0",
-        "symfony/http-kernel": "~3.0||^4.0||^5.0",
-        "symfony/property-access": "~3.0||^4.0||^5.0",
-        "symfony/yaml": "~3.0||^4.0||^5.0"
+        "symfony/console": "^3.4 || ^4.0 || ^5.0 || ^6.0",
```
Comment (maxbaldanza) [inline composer.json:21]: Can you remove support for 3.4 please as `getRootNode` will not exist and will throw an exception

File: README.md [hunk 2]
```diff
@@ -50,14 +50,7 @@ my_builder_cronos:
         shell: /bin/bash
 ```
 
-option   | description
----------|-----------------------------------------
 key      | Unique key that wraps all the cron configured for the current application.
-mailto   | Sets the default email address for all cron output to go to.
```
Comment (maxbaldanza) [inline README.md:56]: Why were these options removed from the docs?

Comment (Chris53897) [reply README.md:56]: This looks like a bug in the PhpStorm function something like "Fix missing outter table pipes | ". Sorry i missed that.

File: composer.json (hunk 1 above)
Comment (Chris53897) [reply composer.json]: done

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Cronos Bundle
 
-A bundle for Symfony 3/4/5 that allows you to use `@Cron` annotations to configure when cron should run your console commands.
+A bundle for Symfony 3/4/5/6 that allows you to use `@Cron` annotations to configure when cron should run your console commands.
```
Comment (maxbaldanza) [inline README.md:3]:
```suggestion
A bundle for Symfony 4/5/6 that allows you to use `@Cron` annotations to configure when cron should run your console commands.
```

File: composer.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "mybuilder/cronos-bundle",
-    "description": "Symfony 3/4/5 bundle which allows you to use @Cron annotations to configure cron to run your console commands.",
+    "description": "Symfony 3/4/5/6 bundle which allows you to use @Cron annotations to configure cron to run your console commands.",
```
Comment (maxbaldanza) [inline composer.json:3]:
```suggestion
    "description": "Symfony 4/5/6 bundle which allows you to use @Cron annotations to configure cron to run your console commands.",
```

File: .gitignore [hunk 5]
```diff
@@ -1,5 +1,6 @@
 composer.lock
 composer.phar
+.phpunit.result.cache
```
Comment (maxbaldanza) [inline .gitignore:3]: We seem to have this twice, can it be removed?

**Review by maxbaldanza — COMMENTED** [review]
Looks great 👍  if we can remove Symfony 3 from docs as it's no longer supported then all good

File: .gitignore (hunk 5 above)
Comment (Chris53897) [reply .gitignore:3]: done

**Comment by Chris53897** [comment]
With the composer type set to `symfony-bundle`, Symfony Flex handle the entry in `config/bundles.php`
I adjusted the README.md

After that is merged, can i create a new PR to drop support of php 7?

**Review by maxbaldanza — APPROVED** [review]
Thank you for the help 👍

**Comment by maxbaldanza** [comment]
> After that is merged, can i create a new PR to drop support of php 7?

Yes happy to do that 👍  thank you

**Pull request merged**